lib/brewdler/utils.rb

```diff
@@ -0,0 +1,10 @@
+module Brewdler
+  def self.system cmd, *args
+    pid = fork do
+      $stdout.reopen("/dev/null")
+      exec(cmd, *args)
+    end
```

Comment:
I did try `IO.popen`. But it may be too nasty for me to handle. I don't know how to use it to handle `stderr` or adding `--verbose` feature in the future. Sorry about this.